composer.json

```diff
@@ -24,14 +24,13 @@
     "zendframework/zend-servicemanager": "^2.7.6 || ^3.1",
     "zendframework/zend-hydrator": "^1.1 || ^2.2.1",
     "zendframework/zend-modulemanager": "^2.7.2",
-    "api-skeletons/zf-doctrine-module-zend-hydrator": "^1.0",
-    "doctrine/doctrine-module": "^1.2",
+    "doctrine/doctrine-module": "^2.1",
```

Comment:
Is there any reason to remove ^1.2 support? Can we keep both here?
Also the same question for `doctrine/doctrine-orm-module`.

Reply:
Yes.  This is a problem I've  been trying to correct for years.  This module cannot support the old Zend/Stdlib/Hydrator and the current.

https://github.com/doctrine/DoctrineModule/issues/546

If you need more links to more discussions on the topic I can provide.

Reply:
Since Zend created zend-hydratror this has been a problem.  Doctrine*Module 2.1 is the release which finally fixed it at the bottom of the requirement stack

Reply:
So, if only support doctrine-module: ^2.1, then why still there above zend-hydrator & zend-servicemanager with unsupported version (because doctrine-module require only strong zend-hydrator ^2.3 &  zend-servicemanager ^3.3).
And BTW, why are they here dublicate requires from doctrine-module?